src/librbd/ObjectMap.h

```diff
@@ -120,6 +122,8 @@ class ObjectMap {
 
   ceph::BitVector<2> m_object_map;
 
+  bool m_enabled;
```

Comment:
Minor: shouldn't be an issue, but probably should initialize to false in constructor